packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -480,11 +476,10 @@ export class SceneTree {
   @Method()
   public async getRowForEvent(event: MouseEvent | PointerEvent): Promise<Row> {
     const { clientY, currentTarget } = event;
-    const rowsEl = this.el.shadowRoot?.querySelector('.rows');
     if (
       currentTarget != null &&
-      rowsEl != null &&
-      getSceneTreeContainsElement(rowsEl, currentTarget as HTMLElement)
```

Comment:
This was evaluating the incorrect element. When adding an event listener to the scene tree, `currentTarget` will be the scene tree and not the child that was clicked.